src/groovy/org/transmartproject/rest/marshallers/DataTypeSerializationHelper.groovy

```diff
@@ -0,0 +1,31 @@
+package org.transmartproject.rest.marshallers
+
+import org.transmartproject.export.DataTypeRetrieved
+
+class DataTypeSerializationHelper  extends AbstractHalOrJsonSerializationHelper<DataTypeRetrieved>{
+
+    final Class targetType = DataTypeRetrieved
+    final String collectionName = 'dataTypeRetrieved'
+
+    @Override
+    Map<String, Object> convertToMap(DataTypeRetrieved dataTypeRetrieved) {
+        def cohortInfoList = []
+        def cohortsMap = [:]
+        dataTypeRetrieved.OntologyTermsMap.each { ID, terms ->
+            terms.collect { term ->
+                if (ID in cohortsMap.keySet()) {
```

Comment:
The frontend wants to know which concepts are in which cohort. Now a cohort can only contain one study, so the most logical thing to do would be to differentiate between studies. However in the near future cross studies will be added, which would result in this serialiser not functioning properly. So there needed to be a way to easily differentiate between the cohorts. This way it also supports multiple cohorts, which is also planned for the future. So based on (cohort)ID I create the structure for cohorts.